structuredheadings/dialogs/selectstyle.js

```diff
@@ -0,0 +1,35 @@
+CKEDITOR.dialog.add("selectStyle", function (editor) {
+
+  var styleOptions = [["Default", null]];
+
+  //Build out the style options
+  (function () {
+    for (var style in CKEDITOR.config.autonumberStyles) {
+      styleOptions.push([style, style]);
```

Comment:
Does this correctly handle defaults?  For example, if CKEDITOR.config.autonumberStyles is not defined in the config.  Is the plugin guaranteed to have put these defaults in by the time this file loads?